status/private/callrpc.nim

```diff
@@ -85,13 +85,14 @@ proc newWeb3*(settings: Settings): Web3 {.raises: [CatchableError, Defect,
 
 proc callRpc*(web3Conn: Web3, rpcMethod: RemoteMethod, params: JsonNode):
   Future[Response] {.async, raises: [Defect, CatchableError].} =
-  const errorMsg = "Error calling RPC method"
   try:
     return await web3Conn.provider.call($rpcMethod, params)
   except CatchableError as e:
-    raise (ref Web3Error)(parent: e, msg: errorMsg)
+    let err = getCurrentExceptionMsg()
+    raise (ref Web3Error)(parent: e, msg: getCurrentExceptionMsg())
```

Comment:
Is there a problem with doing `msg: err` since in the line above you `let err = getCurrentExceptionMsg()`?

Also, I'm curious why it's important in this case (and below) to do `getCurrentExceptionMsg()` instead of referring to `e.msg`?

Reply:
It's a mistake. I'll fix it